lib/services/github_service.rb

```diff
@@ -208,13 +209,13 @@ def create_ref_if_necessary(ref_name, master_head_sha)
     # +ref_sha+:: the sha of the ref to fetch
     def get_ref_name_by_sha(ref_sha)
       client = create_octokit_client
-      ref_response = client.refs(full_repo_name).find { |x| x[:object][:sha] == ref_sha }
+      ref_response = client.refs(@full_repo_name).find { |x| x[:object][:sha] == ref_sha }
       ref_response[:ref].match(/refs\/(.*)/).captures.first
     end
 
     private
       def full_repo_name
```

Comment:
It's not necessary to have this method here anymore. It looks like you started to do this but can you please remove this method and replace it with the full_repo_name instance variable.

Reply:
Yes, makes sense :)